packages/web-components/src/components/cbp-toast/cbp-toast.stories.tsx

```diff
@@ -22,11 +22,12 @@ export default {
     },
   };
   
-  const Template = ({ icon, content, buttons, timer, color, context, sx }) => {
+  const Template = ({ open, icon, content, buttons, duration, color, context, sx }) => {
     return ` 
           <cbp-toast
+            ${open ? `open=${open}` : ''}
             color=${color}
-            timer=${timer}
+            duration=${duration}
```

Comment:
you need to use a ternary for these anytime they are able to be null. Right now anyone copying the code will get these attributes, which is not idea:
  color="undefined"
  duration="undefined"

Reply:
updated to use ternaries as discussed